scripts/serverless.js

```diff
@@ -2,6 +2,9 @@
 
 'use strict';
 
+// `EvalError` is used to not pollute global namespace but still have the value accessible globally
```

Comment:
We should do it in `bin/serverless.js` (as it should also cover Components Triage logic)